apps/src/javalab/Javalab.js

```diff
@@ -148,19 +150,9 @@ Javalab.prototype.onContinue = function() {
   });
 };
 
-Javalab.prototype.getCodeAsync = function() {
-  return new Promise((resolve, reject) => {
-    onSave(
-      /* success */
-      () => resolve(this.getCurrentFilesVersionId() || ''),
-      /* failure - couldn't save files, don't try to overwrite any sources */
-      reject
-    );
-  });
-};
-
-Javalab.prototype.getCurrentFilesVersionId = function() {
-  return project.filesVersionId;
+Javalab.prototype.getCode = function() {
```

Comment:
@madelynkasula you had mentioned in the doc using `sources` as a variable instead of `source`. This PR is sticking with `source` because the project code automatically stores what is returned by `getCode` into `source`. If we wanted to switch this we'd probably need a special case for Javalab or we would need to rework it for all the labs, which I worry would cause a lot of unforeseen issues. Do you feel strongly about using the plural instead of the singular?

Reply:
no, i think you made the right choice -- if there's anything already set up to expect `source`, we should stick with that. i had thought it might be clearer (e.g., `source` will only ever contain 1 source file and `sources` will contain multiple), but consistency is probably better here, and we can always check the type of the `source` value to determine if it's multi- or single-source